source/expressions.tex

```diff
@@ -6248,6 +6248,9 @@
 
 \end{itemize}
 
+When comparing two unequal pointers to functions,
```

Comment:
Hm, this is a somewhat unusual way of saying things. How would you feel about the usual "The result of comparing two unequal pointers is unspecified"?

Reply:
This is a bit of a tricky two-stage process. We first define "(is required to) compare greater" and then we define the results of the comparison operators based on that (partial) order. But that's only for pointer to object.

I've updated the second sentence where the results of the actual relational operators are defined. This feels better now.

Reply:
Agreed, this is nice. Thanks!